### PR TITLE
Add prismic-dom types for version 2.1

### PR DESCRIPTION
**Pull request by douglasnomizo**
- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Add or edit tests to reflect the change. (Run with `npm test`.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/prismicio/prismic-dom/blob/master/src/richtext.js
- [X] Increase the version number in the header if appropriate.
- [X] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/prismic-dom/index.d.ts [hunk 1]
```diff
@@ -15,6 +53,7 @@ interface Link {
 
 export const RichText: RichText;
 export const Link: Link;
+export const HTMLSerializer: HTMLSerializer<string>;
 
-declare const _default: { RichText: RichText, Link: Link };
+declare const _default: {RichText: RichText; Link: Link};
```
Comment (sbilstein) [inline types/prismic-dom/index.d.ts]: why is this a semicolon now?

File: types/prismic-dom/index.d.ts [hunk 2]
```diff
@@ -1,12 +1,50 @@
-// Type definitions for prismic-dom 2.0
+// Type definitions for prismic-dom 2.1
 // Project: https://github.com/prismicio/prismic-dom#readme
 // Definitions by: Nick Whyte <https://github.com/nickw444>
 //                 Siggy Bilstein <https://github.com/sbilstein>
+//                 Douglas Nomizo <https://github.com/douglasnomizo>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
+
+type ElementType =
```
Comment (sbilstein) [inline types/prismic-dom/index.d.ts]: It looks like `prismic-rich-text`has some type definitions already. Did you consider including those somehow? 
https://github.com/prismicio/prismic-richtext/blob/master/d.ts/elements.d.ts

**Comment by sbilstein** [comment]
Just some minor questions. I'm not an expert in writing type definitions; more just a consumer of them but I'll take a look again once you get back to us.

File: types/prismic-dom/index.d.ts (hunk 2 above)
Comment (douglasnomizo) [reply types/prismic-dom/index.d.ts]: It's actually a simplified version of the definition in the `prismic-richtext` library: https://github.com/prismicio/prismic-richtext/blob/b4822be30aa691e9d66dbeb8d215ae957c95d06a/src/serialize.ts#L5

I also included a list of element types

File: types/prismic-dom/index.d.ts (hunk 1 above)
Comment (douglasnomizo) [reply types/prismic-dom/index.d.ts]: Just trying to be more consistent with other packages (e.g. [node-xlsx](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node-xlsx/index.d.ts#L31-L35))

Indentation was updated

File: types/prismic-dom/index.d.ts (hunk 2 above)
Comment (sbilstein) [reply types/prismic-dom/index.d.ts]: Makes sense.

**Comment by sbilstein** [comment]
lgtm

**Review by sbilstein — APPROVED** [review]

**Comment by rbuckton** [comment]
Thanks for the contribution!

**Pull request merged**